src/main/src/main/java/org/geoserver/ows/LocalWorkspaceCallback.java

```diff
@@ -23,7 +23,7 @@
 
 /**
  * Dispatcher callback that sets and clears the {@link LocalWorkspace} and {@link LocalPublished}
- * thread locals.
+ * thread locals. "abc", "abc/xyz"
```

Comment:
I don't get the intent of this change.

Reply:
it took me some time to understand the thread local context was of the form "workspace/layer", or "layergroup" or whatever ...

I will improve the javadoc while I wait for a build.